packages/datatrak-web/src/components/Toast/utils/useToast.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import { enqueueSnackbar } from 'notistack';
+import { Coconut, Pig } from '../../Icons';
+import { errorToast } from '.';
+
+/**
+ * Utility hook for displaying toast messages.
+ */
+export const useToast = () => {
```

Comment:
This doesn't actually need to be a custom hook since there is no useState or useEffect in it. We could just export each of these as a util function or maybe we don't even need them since the pig and coconut will probably only be used in one place.